pom.xml

```diff
@@ -144,13 +144,18 @@
     </dependency>
     <dependency>
       <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-log4j12</artifactId>
+      <artifactId>slf4j-simple</artifactId>
```

Comment:
Why do we need `sl4j-simple` if we depend on `log4j2` ?

Reply:
Thanks for the feedback @tsegismont . I updated my commit to exclude `slf4j-simple` in `vertx-zookeeper` and I put it in my application instead. 
If I don't include islf4j-simple in either application, I get the no-operation logger implementation errors, but I guess that is to be expected.